tests/lib/rules/use-isnan.js

```diff
@@ -349,140 +349,456 @@ ruleTester.run("use-isnan", rule, {
     invalid: [
         {
             code: "123 == NaN;",
-            errors: [comparisonError]
+            errors: [{
+                ...comparisonError,
+                suggestions: [
+                    {
+                        messageId: "replaceWithIsNaN",
+                        output: "Number.isNaN(123);"
+                    },
+                    {
+                        messageId: "replaceWithCastingAndIsNaN",
+                        output: "Number.isNaN(Number(123));"
+                    }
+                ]
+            }]
         },
         {
             code: "123 === NaN;",
-            errors: [comparisonError]
+            errors: [{
+                ...comparisonError,
+                suggestions: [
+                    {
+                        messageId: "replaceWithIsNaN",
+                        output: "Number.isNaN(123);"
+                    },
+                    {
+                        messageId: "replaceWithCastingAndIsNaN",
+                        output: "Number.isNaN(Number(123));"
+                    }
+                ]
+            }]
         },
```

Comment:
For `===` or `!==`, I think there should be only one suggestion (without `Number()`).

Reply:
Ohh, yeah, this makes more sense. No I get what you mean.
Done :)